lib/matplotlib/text.py

```diff
@@ -1158,14 +1159,19 @@ def set_verticalalignment(self, align):
 
     def set_text(self, s):
         """
-        Set the text string *s*
+        Set the text string *s*.
 
         It may contain newlines (``\\n``) or math in LaTeX syntax.
 
-        ACCEPTS: string or anything printable with '%s' conversion.
+        ACCEPTS: string or anything printable with '%s' conversion, except
```

Comment:
"anything printable with '%s' conversion" is technically correct, but feels a bit beside the point. This has nothing to do with printing. I don't really know a good alternative.

According to `__str__` docs "anything with an "informal" string representation" would be right, but I'm afraid that users will not understand this.

Maybe "anything that can be formatted by '%s'" or "anything that can be converted by '%s'"?

Reply:
Actually any object has a `__str__`... (yes, you can technically make `__str__` raise an exception but whatever).
The test should more be something like "any object, will be cast to a string using `str`" (don't care much for Py2 now especially if this goes to mpl3.0...).